libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -273,6 +274,22 @@ const decorateColumns = async ({ content, separatorTagName = 'H5' } = {}) => {
   }
 };
 
+const decorateCrossCloudMenu = (content) => {
+  const crossCloudMenuEl = content.querySelector('.cross-cloud-menu');
+  if (!crossCloudMenuEl) return;
+
+  const crossCloudMenuContent = toFragment`<div class="feds-crossCloudMenu"></div>`;
+
+  [...crossCloudMenuEl.querySelector('ul').children].forEach((el, index) => {
+    const decoratedLink = decorateLinkGroup(el, index);
```

Comment:
Continuing the previous comment, if an author forgets to add a link and leaves a simple text list item, then this will return an empty string. Trying to prepend things to it in the next step might result in issues